packages/snaps-execution-environments/src/common/BaseSnapExecutor.test.ts

```diff
@@ -323,7 +323,7 @@ describe('BaseSnapExecutor', () => {
                 }, 1000);
                 return promise;
               case 'getHandle':
-                return handle;
+                return {};
```

Comment:
The test could work if rewritten - see my comment

Reply:
I've removed this test as requested from the team and @ritave specifically here. 
I read your comment now but I also think that because we are prohibiting anything that is `unhardened endowment`, we don't have to do it in the tests. Perhaps it's alright to have it removed?

Reply:
This test was (incorrectly) testing a valid feature. IMHO it deserves keeping, rewritten to actually work. 
The idea is to pass the handle by setting it as a field of an unhardened endowment that both snaps get.

Reply:
This is getting into diminishing returns territory. Nuke it for now

Reply:
The final agreement is to remove the test as it is now on this PR. I will resolve this and related review threads on this PR.